topics/single-cell/index.md

```diff
@@ -13,6 +13,6 @@ Check out workflows shared by users like you!
 
 If you want to help us behind the scenes, from testing workflows and tutorials to building tools, join our Galaxy Single Cell Community of Practice!
 
- - <i class="fa fa-hand-o-right" aria-hidden="true"></i> [Community of Practice](https://galaxyproject.org/projects/singlecell/)
- - <i class="fa fa-comments-o" aria-hidden="true"></i> [Matrix Chat Forum](https://matrix.to/#/#usegalaxy-eu_single-cell-workflows:gitter.im)
+ - <i class="fa fa-hand-o-right" aria-hidden="true"> </i> [Community of Practice](https://galaxyproject.org/projects/singlecell/)
+ - <i class="fa fa-comments-o" aria-hidden="true"> </i> [Matrix Chat Forum](https://matrix.to/#/#usegalaxy-eu_single-cell-workflows:gitter.im)
  - <i class="fa fa-envelope" aria-hidden="true"> </i> [Mailing List](https://lists.galaxyproject.org/lists/single-cell-cop.lists.galaxyproject.org/)
```

Comment:
`{% icon .. %}` should function here, let's add support for whatever icons you need to `_config.yml` (just give them nice human names) to make it a bit more readable for you and others?

```suggestion
 - <i class="fa fa-hand-o-right" aria-hidden="true"> </i> [Community of Practice](https://galaxyproject.org/projects/singlecell/)
 - {% icon feedback %} [Matrix Chat Forum](https://matrix.to/#/#usegalaxy-eu_single-cell-workflows:gitter.im)
 - <i class="fa fa-envelope" aria-hidden="true"> </i> [Mailing List](https://lists.galaxyproject.org/lists/single-cell-cop.lists.galaxyproject.org/)
```

Reply:
like this?   point-right: fa fa-hand-o-right

Reply:
yep! fas / far are 'solid' and 'regular', whichever one is fitting better + free (if you check the fontawesome website/search)